pages/franchise_page/[franchise].md

```diff
@@ -0,0 +1,200 @@
+and slot != 'NONE'
+and season_number = 17
+order by slot asc
+```
+
+```sql players
+select 
+name,
+'/player_page/' || p.member_id as id_link,
+salary,
+skill_group as league,
+franchise,
+SUBSTRING(slot, 7) as slot,
+doubles_uses,
+standard_uses,
+total_uses,
+current_scrim_points,
+case when current_scrim_points >= 30 then 'Yes'
+    else 'No'
+    end as Eligible
+from players p
+    inner join role_usages ru
+        on p.franchise = ru.team_name
+        and p.slot = ru.role
+        and upper(p.skill_group) = concat(ru.league, ' LEAGUE')
+where franchise = '${params.franchise}'
+and skill_group = '${inputs.League.value}'
+and slot != 'NONE'
+and season_number = 17
+order by slot asc
+```
+
+> League Selection
+<Dropdown data={player_info} name=League value=league />
+
+> Franchise Players
+<DataTable data={players} rowshading=true headerColor='{team_info[0].primary_color}' headerFontColor=white wrapTitles=true>
+    <Column id=id_link contentType=link linkLabel=name align=center title=Player />
+    <Column id=salary align=center />
+    <Column id=slot align=center />
+    <Column id=doubles_uses align=center contentType=colorscale scaleColor={['#6db678','white','#ce5050']} colorMin=0 colorMid=3 colorMax=6 />
+    <Column id=standard_uses align=center contentType=colorscale scaleColor={['#6db678','white','#ce5050']} colorMin=0 colorMid=4 colorMax=8 />
+    <Column id=total_uses align=center contentType=colorscale scaleColor={['#6db678','white','#ce5050']} colorMin=0 colorMid=6 colorMax=12 />
+    <Column id=current_scrim_points align=center />
+</DataTable>
+
+```sql gamemodes
+select game_mode
+from matches
+```
+
+```sql team_record
+with record as(
+select 
+home,
+away,
+league,
+game_mode,
+home_wins,
+away_wins,
+winning_team as series_winner,
+parent_group_title,
+game_mode,
+match_group_title
+from matches m
+    inner join match_groups mg
+        on m.match_group_id = mg.match_group_id
+    where parent_group_title = 'Season 17'
+    and home = '${params.franchise}'
+    and league = '${inputs.League.value}'
+    and game_mode = '${inputs.Gamemodes.value}'
+    or parent_group_title = 'Season 17'
+    and away = '${params.franchise}'
+    and league = '${inputs.League.value}'
+    and game_mode = '${inputs.Gamemodes.value}'
+order by m.match_group_id asc
+)
+select 
+SUBSTRING(match_group_title, 7) as week,
+home,
+away,
+series_winner,
+CASE WHEN series_winner = '${params.franchise}' THEN 'Win' 
+    WHEN series_winner = 'Not Played / Data Unavailable' THEN 'NA'
+        ELSE 'Loss' 
+            END AS series_result,
+CASE WHEN home = '${params.franchise}' THEN concat(cast(home_wins as integer), ' - ', cast(away_wins as integer))   
+    WHEN away = '${params.franchise}' THEN concat(cast(away_wins as integer), ' - ', cast(home_wins as integer))
+         END as record,
+series_winner,
+game_mode
+from record
+```
+
+>GameMode Selection
+<Dropdown data={gamemodes} name=Gamemodes value=game_mode />
+
```

Comment:
Add a [Multiple Card Big Value](https://docs.evidence.dev/components/big-value/#multiple-cards) here that shows the team's record, series record, and goal diff